src/xcode/ENA/ENA/Source/Services/Risk/Provider/Helper/__tests__/CachedAppConfigurationTests.swift

```diff
@@ -84,6 +84,41 @@ final class CachedAppConfigurationTests: XCTestCase {
 
 		subscription.cancel()
 	}
+	
+	func testClientMetadata_isUpdated_everytime_appconfiguration_isFetched() {
+		let store = MockTestStore()
+		XCTAssertNil(store.clientMetadata, "Client metadata should be initially nil")
+		let client = CachingHTTPClientMock(store: store)
+		let cache = CachedAppConfiguration(client: client, store: store)
+		let expectationClientMetadata = expectation(description: "ClientMetadata")
+		//AppVersion
+		let appVersionParts = Bundle.main.appVersion.split(separator: ".")
+		guard appVersionParts.count == 3,
+			  let majorAppVerson = Int(appVersionParts[0]),
+			  let minorAppVerson = Int(appVersionParts[1]),
+			  let patchAppVersion = Int((appVersionParts[2])) else {
+			return
+		}
+		let expectedAppVersion = Version(major: majorAppVerson, minor: minorAppVerson, patch: patchAppVersion)
+		//iOSVersion
+		let expectediosVersion = Version(
+			major: ProcessInfo().operatingSystemVersion.majorVersion,
+			minor: ProcessInfo().operatingSystemVersion.minorVersion,
+			patch: ProcessInfo().operatingSystemVersion.patchVersion
+		)
+		//eTag
+		let expextedETag = "fake"
+		let configuration = cache.appConfiguration(forceFetch: true).sink { _ in
+			expectationClientMetadata.fulfill()
+		}
+		waitForExpectations(timeout: 1) { _ in
```

Comment:
Let's use the (longer) constants. We have enough flaky tests:
```suggestion
		waitForExpectations(timeout: .medium) { _ in
```